syntax = "proto3";

service Microservices {
    rpc ListFiles (ListRequest) returns (FilesResponse);
    rpc SearchFiles (SearchRequest) returns (FilesResponse);
}

message ListRequest {}

message SearchRequest {
    string pattern = 1;
}

message FilesResponse {
    repeated string files = 1;
}
